spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/IRGraph.scala

```diff
@@ -55,28 +56,44 @@ private[bigdl] class IRGraph[T: ClassTag](
 
   private var graph: Graph[T] = null
 
+  private[bigdl] def isBuild(): Boolean = if (graph == null) false else true
```

Comment:
change to 

`private[bigdl] def isBuild(): Boolean = graph != null`

Reply:
ok, changed